src/web/interfaces.ts

```diff
@@ -121,22 +122,19 @@ export interface AdaptedEvent {
   pointerId: number;
   eventType: EventTypes;
   pointerType: PointerType;
-  buttons: number;
   time: number;
+  button?: MouseButton;
   allTouches?: TouchList;
   changedTouches?: TouchList;
   touchEventType?: TouchEventType;
 }
 
-export enum MouseButtons {
-  NONE,
-  LEFT,
-  RIGHT,
-  LEFT_RIGHT,
-  SCROLL,
-  SCROLL_LEFT,
-  SCROLL_RIGHT,
-  SCROLL_LEFT_RIGHT,
+export enum MouseButton {
+  LEFT = 1,
+  MIDDLE = 2,
+  RIGHT = 4,
+  BTN4 = 8,
+  BTN5 = 16,
```

Comment:
```suggestion
  BUTTON_4 = 8,
  BUTTON_5 = 16,
```

I would go for more expressive names, we have autocompletion anyway.

Reply:
I was used to these abbreviations, but renaming it to full names also sounds good. Done in [193e240](https://github.com/software-mansion/react-native-gesture-handler/pull/2676/commits/193e24009e37266cd1a9ec800c3525a91627c889)